src/telegram.js

```diff
@@ -490,6 +490,28 @@ class TelegramBot extends EventEmitter {
     form.longitude = longitude;
     return this._request('sendLocation', { form });
   }
+  
+  /**
+   * Send Venue.
+   * Use this method to send Venue infomration
+   *
+   * @param  {Number|String} chatId  Unique identifier for the message recipient
+   * @param  {Float} latitude Latitude of venue
+   * @param  {Float} longitude Longitude of venue
+   * @param  {String} title Title of venue
+   * @param  {String} address Address of venue
+   * @param  {Object} [options] Additional Telegram query options
+   * @return {Promise}
+   * @see https://core.telegram.org/bots/api#sendlocation
+   */
+  sendVenue(chatId, latitude, longitude, title, address, form = {}) {
+    form.chat_id = chatId;
+    form.latitude = latitude;
+    form.longitude = longitude;
+    form.title = title;
+    form.address = address;
+    return this._request('sendVenue', { form });
```

Comment:
Why you need form object if you send directly values that you provide in argument of function ???
just
` return this._request('sendVenue', { 
chatId,
latitude,
longitude,
title,
address
 });`
keep it simple =)